terraform/aws/README.md

```diff
@@ -57,7 +57,7 @@ variable like so:
 
 ```bash
 region                  = "us-east-1"
-ami                     = "ami-066a7f2ffac02f833"
+ami                     = "ami-05847887cacd41419"
```

Comment:
What changed in the ami?  Was it a routine refresh as I don't quite see packer related changes here.

Reply:
So the [user_data](https://github.com/hashicorp/nomad/tree/master/terraform/aws/env/us-east) scripts call the [setup](https://github.com/hashicorp/nomad/tree/master/terraform/shared/scripts) which are packed into the image. This is where I made sure the AWS CLI is installed and incorporated some [logic](https://github.com/hashicorp/nomad/blob/nomad-tf-config-ent/terraform/shared/scripts/client.sh#L45-L49) for pulling down the enterprise binaries form an S3 bucket.